README.md

```diff
@@ -57,18 +57,61 @@ Be sure to properly configure your environment for the USDC and YFI tokens on Et
 To run your bot against a live network, this SDK includes a simple runner command you can use via:
 
 ```sh
-$ silverback run "example:app" --network :mainnet:alchemy
+$ silverback run example --network :mainnet:alchemy
 ```
 
 ```{note}
 This runner uses an in-memory task broker by default.
 If you want to learn more about what that means, please visit the [development userguide](https://docs.apeworx.io/silverback/stable/userguides/development.html).
 ```
 
+```{note}
+It is suggested that you create a bots/ folder in the root of your project.
+Silverback will automatically register files in this folder as separate bots that can be run via the `silverback run` command.
+
+It is also suggested that you treat this as a scripts folder, and do not include an __init__.py
+If you have a complicated project, follow the previous example to ensure you run the application correctly.
+
+A final suggestion would be to name your `SilverbackApp` object `bot`. Silverback automatically searches 
+for this object name when running. If you do not do so, once again, ensure you replace `example` with 
+`example:<name-of-object>` the previous example.
+```
+
+To auto-generate Dockerfiles for your bots, from the root of your project, you can run:
+
+```bash
+silverback build
+```
+
+This will place your dockerfiles in the root of your project.
+
+As an example, if you have a bots directory that looks like:
+
+```
+bots/
+├── botA.py
+├── botB.py
+├── botC.py
+```
+
+This method will generate 3 Dockerfiles:
+
+```
+├── Dockerfile.botA
+├── Dockerfile.botB
+├── Dockerfile.botC
+```
+
+These Dockerfiles can be deployed with the `docker run` command documented in the next section.
+
+```{note}
+As an aside, if your bots/ directory is a python package, you will cause conflicts with the dockerfile generation feature. This method will warn you that you are generating runners for a python package, but will not stop you from doing so. If you choose to generate dockerfiles, the user should be aware that it will only copy each individual file into the Dockerfile, and will not include any supporting python functionality. Each python file is expected to run independently. If you require more complex runners, you will have to build a custom docker image.
```

Comment:
suggestion: Maybe don't use the term "runners". That's got other meaning within Silverback. "Workers" , "bots", or "dockerfiles" might be more appropriate?

thought: I wonder if it's appropriate to tell the users that these are just templates and it's a good idea to add instructions for things like "supporting python functionality" rather than telling them it's a bad thing.